proofs.go

```diff
@@ -414,6 +414,13 @@ func SealCommitPhase2(
 	return cgo.SealCommitPhase2(cgo.AsSliceRefUint8(phase1Output), uint64(sectorNum), &proverID)
 }
 
+// SealCommitPhase2CircuitProofs runs a non-interactive proof and returns the circuit proof bytes
+// rather than the aggregated proof bytes. This is used to aggregate multiple non-interactive
+// proofs as the aggregated single proof outputs can't further be aggregated.
+func SealCommitPhase2CircuitProofs(phase1Output []byte, sectorNum abi.SectorNumber) ([]byte, error) {
+	return cgo.SealCommitPhase2CircuitProofs(cgo.AsSliceRefUint8(phase1Output), uint64(sectorNum))
+}
```

Comment:
@vmx do you think I describe this properly here?

Reply:
Yes, sounds good.